ods_ci/tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot

```diff
@@ -54,3 +55,37 @@ Get Query Results
         END
     END
     RETURN  ${metrics_query_result_row1_value}
+
+Run OpenShift Metrics Query
+    [Documentation]    Runs a query in the Monitoring section of Open Shift
+    ...    Note: in order to run this keyword OCP_ADMIN_USER.USERNAME needs to
+    ...    belong to a group with "view" role in OpenShift
+    ...    Example command to assign the role: oc adm policy add-cluster-role-to-group view rhods-admins
+    [Arguments]    ${query}    ${retry_attempts}=10    ${return_zero_if_result_empty}=False   ${usertype}=admin
```

Comment:
What if, instead of having `${usertype}=admin` we have parameters `${user}=${OCP_ADMIN_USER.USERNAME} and ${password}=${OCP_ADMIN_USER.PASSWORD}`

Reply:
+1